tools/API_SHAS

```diff
@@ -1,11 +1,11 @@
 # Update the versions here and run update-api.sh
 
 # envoy (source: SHA from https://github.com/envoyproxy/envoy)
-ENVOY_SHA="38fe5e1ec115c0b2e83921211c2646f160a91681" # Feb 13, 2020
+ENVOY_SHA="8fb3cb86082b17144a80402f5367ae65f06083bd"  # Oct 8, 2020
```

Comment:
Maybe it is worth to add `(v1.16.0)` to the comment?

Reply:
added